src/stories/SideNavigation.stories.tsx

```diff
@@ -0,0 +1,165 @@
+import React, { useState } from 'react'
+import {
+  Avatar,
+  Button,
+  NavigationItem,
+  SideNavigation as SideNavigationItemComp,
+  Text
+} from '../components'
+import Block from '../components/SideNavigation/Block'
+import logo from '../assets/images/logo.svg'
+import logoClosed from '../assets/images/logo-closed.svg'
+import avatar from '../assets/images/avatar.jpg'
+
+export default {
+  title: 'Side Navigation',
+  component: SideNavigationItemComp,
+  argTypes: {
+    showAction: {
+      options: [true, false]
+    }
+  }
+}
+
+const Template = (args): JSX.Element => {
+  const [open, setOpen] = useState<boolean>(true)
+
+  return (
+    <SideNavigationItemComp setOpen={setOpen} open={open} {...args}>
+      <main>
+        <NavigationItem
+          As={() => (
+            <a href="/https://ameriabank.am">
+              <img src={open ? logo : logoClosed} alt="" />
+            </a>
+          )}
+          type={'block-header'}
+          open={open}
+          showAction={args.showAction}
+          actionElm={args.actionElm}
+        />
+        <Block open={open} label="Finance">
+          <>
+            <NavigationItem
+              As={() => <a href="/https://ameriabank.am">Accounts</a>}
+              type={'main'}
+              iconName="document"
```

Comment:
Let's have code consistency and if it's not a variable not to use curly braces for assigning value

Reply:
I already created enum for this types and fix  it, but I agree with you